### ISSUE TITLE
Sticky header floating under top when there is no buttons in the toolbar

### ISSUE THREAD

**Issue by OleksiiBulba** [magento/magento2#23230]
### Preconditions (*)
1. Install Magento 2.3.1
2. Create module Vendor_Module

### Steps to reproduce (*)
1. Create listing grid without this node:
```
<listing ...>
    <settings>
        <!--<buttons></buttons>-->
    </settings>
</listing>
```
2. Set this node value:
```
<listing ...>
    <listingToolbar>
        <settings>
            <sticky>true</sticky>
        </settings>
    </listingToolbar>
</listing>
```
3. Open admin grid page and scroll down for toolbar sticking to top

### Expected result (*)
1. Toolbar sticks to top:
![Magento_Issue_Sticky_Header_Expected](https://user-images.githubusercontent.com/41155673/59355671-17dfcb80-8d30-11e9-9961-0959cb09d755.png)

### Actual result (*)
1. Toolbar sticks to 77px under the top
![Magento_Issue_Sticky_Header](https://user-images.githubusercontent.com/41155673/59355692-20380680-8d30-11e9-80d3-da92c07aacdd.png)

**Comment by OleksiiBulba** [comment]
This issue related to #8455

**Comment by konarshankar07** [comment]
Hello @OleksiiBulba 
I've fixed this issue and updates are on https://github.com/magento/magento2/pull/23247. Please test and let me know if you find any problem.

Thanks

**Comment by shikhamis11** [comment]
Hello @OleksiiBulba 
Thanks for your report
this issue seems to be duplicate of #22614 so closing this one

**Comment by magento-engcom-team** [comment]
Hi @OleksiiBulba. Thank you for your report. 
The issue has been fixed in magento/magento2#23247 by @konarshankar07 in 2.3-develop branch
Related commit(s):
 - [ce56c70e90172d8cb936c12232bf66532ea8b32c](https://github.com/magento/magento2/commit/ce56c70e90172d8cb936c12232bf66532ea8b32c)
 - [f5a93b779a41a5cb14830814e553876ccb7cdbcb](https://github.com/magento/magento2/commit/f5a93b779a41a5cb14830814e553876ccb7cdbcb)
 - [a3fa8d1e2cb5713c7071c0fa8997af57fe67a559](https://github.com/magento/magento2/commit/a3fa8d1e2cb5713c7071c0fa8997af57fe67a559)

The fix will be available with the upcoming 2.3.3 release.
